docs/src/routes/docs/components/checkbox-chip-group.svx

```diff
@@ -0,0 +1,101 @@
+---
+name: CheckboxChipGroup
+relatedComponents:
+  - name: Chip
+    link: ./docs/components/chip
+  - name: RadioChip
+    link: ./docs/components/radio-chip
+  - name: RadioChipGroup
+    link: ./docs/components/radio-chip-group
+  - name: CheckboxChip
+    link: ./docs/components/checkbox-chip
+---
+
+<script>
+  import { CheckboxChipGroup } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+  const items = [
+    { value: 1, label: 'option 1', checked: true },
+    { value: 1.5, label: 'option 1.5', disabled: true },
+    { value: 2, label: 'option 2' },
+  ];
+</script>
+
+# CheckboxChipGroup
+
+A set of options presented as chips with multiple options to select.
+
+If you need to allow selecting only one option, consider using [==`RadioChipGroup`==](./docs/components/radio-chip-group). If you need more customization for the chips, such as their slots, take a look at [==`CheckboxChip`==](./docs/components/checkbox-chip), the building block of this component.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <CheckboxChipGroup {items} name="group1" />
+    <CheckboxChipGroup {items} name="group2" outline />
+    <CheckboxChipGroup {items} name="group3" small />
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  const items = [
+    { value: 1, label: 'option 1', checked: true },
+    { value: 1.5, label: 'option 1.5', disabled: true },
+    { value: 2, label: 'option 2' },
+  ];
+</script>
+
+<CheckboxChipGroup {items} name="group1" />
+<CheckboxChipGroup {items} name="group2" outline />
+<CheckboxChipGroup {items} name="group3" small />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+Any properties of `CheckboxChip` can be specified – they will be passed down.
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| ==**`items`**==&nbsp;<span class="required">*</span> |  | `Array` | An array of `{ value, label, disabled, checked }` objects, where only the value is required. Must contain at least one value. |
+| ==**`name`**==&nbsp;<span class="required">*</span> |  | `string` | The name of this checkbox group. |
```

Comment:
How about a note specifying that it should be unique from other groups? Or even a link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefname

Reply:
It doesn't have to be unique, maybe having two groups with the same name is the developer's intention. As for that link, I really don't see a need to educate users on the DOM basics

Reply:
I know it doesn't have to be, but the developer may not be aware of what this prop really means and the implications of reusing the same `name`